container_pipeline/utils.py

```diff
@@ -1,10 +1,14 @@
+import hashlib
 import os
 
 
 def get_job_name(job_details):
     """Get jenkins job name from job_detials"""
-    return str(job_details['appid']) + "-" + str(job_details['jobid']) + "-" \
-        + str(job_details['desired_tag'])
+    namespace = str(job_details['appid']) + "-" + str(job_details['jobid']) + \
+        "-" + str(job_details['desired_tag'])
+    project = hashlib.sha224(namespace).hexdigest()
+
+    return project, namespace
```

Comment:
the function name does not really match with its return values?  I dont think we need to make it so complicated. We should just write two different functions in utils.py, one for returning namespace while another for returning the project.